plugins/org/src/components/Cards/OwnershipCard/ComponentsGrid.tsx

```diff
@@ -110,18 +110,21 @@ export const ComponentsGrid = ({
   relationsType,
   isGroup,
   entityFilterKind,
+  showMax = Infinity,
```

Comment:
I think it's a little strange that the `showMax` sounds like a boolean but it is in fact a number. Maybe `entityLimit` or something instead?

Reply:
@benjdlambert thanks for suggestion. Updated it.